app/views/layouts/application.html.erb

```diff
@@ -29,17 +29,17 @@
               <li class="has-dropdown">
                 <a href="#">Profile</a>
                 <ul class="dropdown">
-                  <li>
+                  <li class="active">
```

Comment:
Hey, why did you add `active` to all of the menu items? I think this tells JavaScript that you're currently on this page and only the current page should be `active`.